coral/src/app/features/components/table-filters/MyRequestFilter.tsx

```diff
@@ -0,0 +1,31 @@
+import { Switch } from "@aivenio/aquarium";
+import { useSearchParams } from "react-router-dom";
+
+function MyRequestFilter() {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const isMyRequest = searchParams.get("isMyRequest") === "true" ? true : false;
+
+  const handleChangeIsMyRequest = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    if (event.target.checked) {
+      searchParams.set("isMyRequest", "true");
+    } else {
+      searchParams.delete("isMyRequest");
+    }
+    searchParams.set("page", "1");
+    setSearchParams(searchParams);
+  };
+
+  return (
+    <Switch
+      checked={isMyRequest}
+      onChange={handleChangeIsMyRequest}
+      caption="Only show my requests"
+    >
+      Show My Requests
```

Comment:
We could improve usability for screen reader (at least some) a bit if we wrap the "Show my requests" in an `<span aria-hidden="true">` (if that's possible).

Currently, VO reads: "Show my requests. Only show my requests". Which is a bit redundant. If the span works, it would only be "Only show my requests". 

🤔  On a second thought, maybe it would be even better to not have a caption but have "Only show my requests" as the main text. The caption is only one additional word to the main text, so it does not really add additional value. But that would have to be synced with Design of course.

Reply:
Really good points, thank you. I agree the caption feels redundant. I'll check with Mustafa if we can drop it. Otherwise i'll do the accessibility change you suggested 👍🏼